### DISCUSSION TITLE
Add support for podcast episodes

### DISCUSSION THREAD

**Discussion by lavaldi** [gilbarbara/react-spotify-web-playback#84] [Ideas]
Hi @gilbarbara 👋

I don't know if we can add support for podcast episodes reproduction, I mean, add `episode` in the `validTypes` variable inside `validateURI` 🙏

I have a PR prepared for that (not yet submitted) but I just see the contributing guide, so here is the discussion 😅

Thanks in advance!

**Comment by gilbarbara** [comment]
hey @lavaldi 
I've tried to load episodes a couple of weeks ago but it didn't work.

**Comment by lavaldi** [comment]
I know maybe you did but I'm asking just in case. Did you add the `user-read-playback-position` scope? 👀

**Reply by gilbarbara** [reply]
Can you link the documentation why is it needed for episodes?

**Comment by lavaldi** [comment]
Sorry, I got confused about scope 😅 but it seems that the playback for episodes is not supported yet for the SDK 😞. I tried it [here](https://developer.spotify.com/console/put-play/) and got this error:

```json
{
  "error" : {
    "status" : 400,
    "message" : "Non supported context uri"
  }
}
```

Also, I found [this issue](https://community.spotify.com/t5/Spotify-for-Developers/Episode-uri-not-supported-player-play/m-p/5040833/highlight/true#M1195) in the Spotify for Developers forum.

**Reply by gilbarbara** [reply]
Yep, I've got the same error and found the same issue.
Also, there's a closed issue for it as well: https://github.com/gilbarbara/react-spotify-web-playback/issues/65

**Reply by lavaldi** [reply]
we can only wait... 😞
